src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -529,11 +529,7 @@ public bool ValueTextEquals(ReadOnlySpan<char> text)
             else
             {
                 // Cannot create a span directly since it gets passed to instance methods on a ref struct.
```

Comment:
Seems comment can be removed as well?
```suggestion
                // Cannot create a span directly since it gets passed to instance methods on a ref struct.
```